### PR TITLE
refactor: rename Coinbase to GDAX

### LINKED ISSUE #18: Coinbase Exchange is now GDAX
https://blog.coinbase.com/coinbase-exchange-is-now-gdax-adds-ether-trading-a82cc628aa79#.sy8kxyghb

https://docs.gdax.com/#introduction

> **GDAX transition**
> As part of the rebranding of Coinbase Exchange to GDAX, API endpoints have changed. The old `*.exchange.coinbase.com` endpoints have been discontinued as of Dec 6th, 2016.

### PR DESCRIPTION
**Pull request by jleft**
BREAKING CHANGE: `feedCoinbase` has been renamed to `feedGdax`. Coinbase Exchange has been [rebranded to GDAX](https://blog.coinbase.com/coinbase-exchange-is-now-gdax-adds-ether-trading-a82cc628aa79#.sy8kxyghb). As part of the rebranding of Coinbase Exchange to GDAX, API endpoints have changed. The old `*.exchange.coinbase.com` endpoints have been discontinued as of Dec 6th, 2016.

Fixes #18

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-export { default as feedCoinbase } from './src/coinbase';
+export { default as feedGdax } from './src/gdax';
```
Comment (jleft) [inline index.js]: Not sure what the convention is: should this be `feedGdax` or `feedGDAX`?

**Comment by chrisprice** [comment]
I'd go with feed Gdax -
https://d3fc.io/api/technical-indicator-api.html#macd

File: index.js (hunk 1 above)
Comment (ColinEberhardt) [inline index.js]: Taking the lead from D3 (e.g. `dsvFormat`) , I'd go with `feedGdax`

**Comment by ColinEberhardt** [comment]
Thanks @jleft - I guess this is why BitFlux is showing errors at the moment: http://scottlogic.github.io/BitFlux/master/

**Comment by jleft** [comment]
> Thanks @jleft - I guess this is why BitFlux is showing errors at the moment: http://scottlogic.github.io/BitFlux/master/

Exactly - should be a simple fix (https://github.com/ScottLogic/BitFlux/pull/801) :-)

**Pull request merged**